pacman/pacman-canvas.js

```diff
@@ -1524,11 +1520,7 @@ function geronimo() {
 	}
 
 	function submitEmail() {
```

Comment:
rename the method name?

Reply:
mmm wanted to minimize potential breaking, but can change i guess

Reply:
I'm not going to change this - too many things in the html are referenced by email :/